Table.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Table</title>
+    <style>
+        body{
+            font-family: sans-serif;
+            background-color: aliceblue;
+        }
+        section {
+            position: relative;
+            border: 1px solid #cac6c6;
+            padding-top: 37px;
+            background: #82dede;
+            margin-left: 100px;
+            margin-right: 100px;
+            border-radius: 3px;
+        }
+
+        .container {
+            height: 350px;
+        }
+
+        table {
+            border-spacing: 0;
+            width: 100%;
+        }
+
+        td {
+            border-left: 1px solid #eee;
+        }
+
+        td,
+        th {
+            border-bottom: 1px solid #eee;
+            background: white;
```

Comment:
For colors either have hexadecimal or the string value. Eg either use #fff or white.

Reply:
Done